Code/GraphMol/SmilesParse/smiles.ll

```diff
@@ -359,7 +354,7 @@ s		    {	yylval->atom = new Atom( 16 );
 \%              { return PERCENT_TOKEN; }
 
 [0]		{ yylval->ival = 0; return ZERO_TOKEN; }
-[1-9]		{ yylval->ival = atoi( yytext ); return NONZERO_DIGIT_TOKEN; }
+[1-9]		{ yylval->ival = yytext[0] - '0'; return NONZERO_DIGIT_TOKEN; }
```

Comment:
Totally a Roger change

Reply:
I'm a bit ashamed that I hadn't don't it already!
This was already in smarts.ll